app/assets/javascripts/sufia/app.js

```diff
@@ -12,6 +12,8 @@ Sufia = {
   saveWorkControl: function() {
     var sw = require('sufia/save_work/save_work_control');
     new sw.SaveWorkControl($("#form-progress")).activate();
+      FixedSticky.tests.sticky = false;
```

Comment:
Is this really a property of the save work control, or does it deserve extracting to something generally reusable?

I wonder too about using `#fixedwidget` vs `.fixedsticky`.